modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java

```diff
@@ -147,6 +148,7 @@ public ObjectAction addObjectAction(
 		objectAction.setActive(active);
 		objectAction.setConditionExpression(conditionExpression);
 		objectAction.setDescription(description);
+		_validateDescription(description);
```

Comment:
@Aquiiiles you should call this method along with the other validation methods, as I asked [here](https://github.com/liferay-bpm/liferay-portal/pull/4516#discussion_r2086978101)

```java

_validateExternalReferenceCode(
	externalReferenceCode, 0, objectDefinition.getCompanyId(),
	objectDefinitionId);

_validateDescription(description);
_validateErrorMessage(errorMessageMap, objectActionTriggerKey);
```